extension/src/constants/transaction.ts

```diff
@@ -28,6 +28,7 @@ export enum OPERATION_TYPES {
   revokeTrustlineSponsorship = "Revoke Trustline Sponsorship",
   setOptions = "Set Options",
   setTrustLineFlags = "Set Trustline Flags",
+  bumpFootprintExpiration = "Bump Footprint Expiration",
```

Comment:
missing `restoreFootprint`?

Reply:
yes it is, thanks!

Reply:
done in 05fa8fe00e388f8cbf24ba50b6f75548e8ba489d